examples/involvingstates/example_equiangular_threshold.py

```diff
@@ -0,0 +1,163 @@
+"""
+Equiangular States and the Antidistinguishability Threshold
+===========================================================
+
+In this tutorial, we explore a sharp threshold for the antidistinguishability
+of a special class of quantum states known as equiangular states. We will
+numerically verify a tight bound presented in the paper "Tight bounds for
+antidistinguishability and circulant sets of pure quantum states"
+:footcite:`Johnston_2025_Tight` and visualize the "sharp cliff" where this
+property changes.
+
+This tutorial builds upon the concepts introduced in
+:ref:`sphx_glr_auto_examples_involvingstates_example_state_exclusion.py`.
+"""
+
+# %%
+# The Antidistinguishability Threshold for Equiangular States
+# -----------------------------------------------------------
+#
+# A set of :math:`n` pure states :math:`\{|\psi_0\rangle, \ldots, |\psi_{n-1}\rangle\}`
+# is called *equiangular* if the absolute value of the inner product between
+# any two distinct states is a constant, i.e.,
+# :math:`|\langle \psi_i | \psi_j \rangle| = \gamma` for all :math:`i \neq j`.
+#
+# The paper by Johnston, Russo, and Sikora :footcite:`Johnston_2025_Tight`
+# provides a simple and powerful necessary condition for a set of states to be
+# antidistinguishable.
+#
+# **Corollary 4.2 from** :footcite:`Johnston_2025_Tight`: Let :math:`n \geq 2` be an
+# integer and let :math:`S = \{|\psi_0\rangle, \ldots, |\psi_{n-1}\rangle\}`. If
+#
+# .. math::
+#    |\langle \psi_i | \psi_j \rangle| > \frac{n-2}{n-1}
+#    \quad \forall \ i \neq j,
+#
+# then :math:`S` is **not** antidistinguishable.
+#
+# Crucially, Example 3.3 in the paper demonstrates that this bound is *tight*.
```

Comment:
state what this example is i.e. define the Gram matrix etc. in math mode. Use words to describe (or create a list) to summarize how the numerical verification is supposed to work.